### PR TITLE
Add test for use mapping empty nested path

### PR DESCRIPTION
**Pull request by ericTsiliacos**
## Summary
- add test to ensure nested validator using a rule with an empty error path is mapped to the parent property when used

## Testing
- `./gradlew test --no-daemon` *(fails: Unable to tunnel through proxy)*

------
https://chatgpt.com/codex/tasks/task_e_683f81bb236c832cbe1c449ecd332ec2

### CODE REVIEW

File: src/test/kotlin/validation/dsl/ValidationDslTest.kt [hunk 1]
```diff
@@ -593,4 +593,26 @@ class ValidationDslTest {
...
     }
 
+    @Test
+    fun `use should map empty nested error path to parent`() {
+        data class Leaf(val value: String)
+        data class Wrapper(val leaf: Leaf)
+
+        val leafValidator = validator<Leaf> {
+            root {
+                // add rule without path injection so it retains PropertyPath.EMPTY
```
Comment (ericTsiliacos) [inline src/test/kotlin/validation/dsl/ValidationDslTest.kt:603]:
```suggestion
                // add rule without path injection so it retains PropertyPath.EMPTY
```
